index.html

```diff
@@ -0,0 +1,169 @@
+      <h4 class="main-title">Outfit Catalogue</h4>
+      <div class="line"></div>
+    </div>
+    <!-- First Card -->
+    <div class="card-container">
+      <div class="main-card-container">
+        <div class="card-icon-container">
+          <i class="fas fa-venus"></i>
+        </div>
+        <div class="card-title-container">
+          <p class="card-title">Female</p>
+        </div>
+        <div class="card-description-container">
+          <p class="card-description">Find the female prey in our variety Sara, TopShop, H&M, Mango, American Eagle</p>
+        </div>
+      </div>
+      <!-- Second card -->
+      <div class="main-card-container">
+        <div class="card-icon-container">
+          <i class="fas fa-mars"></i>
+        </div>
+        <div class="card-title-container">
+          <p class="card-title">Male</p>
+        </div>
+        <div class="card-description-container">
+          <p class="card-description">Your favorite outfit in our catalog with variety in Prada, Gucci, Balenciaga</p>
+        </div>
+      </div>
+      <!-- Third card -->
+      <div class="main-card-container">
+        <div class="card-icon-container">
+          <i class="fas fa-star"></i>
+        </div>
+        <div class="card-title-container">
+          <p class="card-title">Ranking</p>
+        </div>
+        <div class="card-description-container">
+          <p class="card-description">Find out about the most controversial and outgoing outfits on our list</p>
+        </div>
+      </div>
+      <!-- Fourth card -->
+      <div class="main-card-container">
+        <div class="card-icon-container">
+          <i class="fas fa-dumbbell"></i>
+        </div>
+        <div class="card-title-container">
+          <p class="card-title">Sporty</p>
+        </div>
+        <div class="card-description-container">
+          <p class="card-description">Unisex sportswear for all occasions Adidas, Asics, Nike, Reebok</p>
+        </div>
+      </div>
+      <!-- Fith card -->
+      <div class="main-card-container">
+        <div class="card-icon-container">
+          <i class="fas fa-info-circle"></i>
+        </div>
+        <div class="card-title-container">
+          <p class="card-title">About</p>
+        </div>
+        <div class="card-description-container">
+          <p class="card-description">Learn more about our work and paid services, contacts, shipments</p>
+        </div>
+      </div>
+    </div>
+    <div class="main-button-container">
+      <a href="#" aria-label="link" class="main-button"> Join Global Outfit <span class="numbers">2022</span></a>
+    </div>
+  </div>
+</section>
+<!-- Project speaker section -->
+<section id="speaker-section">
+  <div id="speaker" class="speaker-container">
+    <div class="main-title-container">
+      <h4 class="main-title-b">Outfit Speakers</h4>
+      <div class="line"></div>
+      </div>
+      <div class="speaker-outfit-card-container"></div>
```

Comment:
- [ ] Good job :+1: in writing this section. I can see you are generating your content dynamically with JavaScript. However, it is essential we try as possible to match the original design layouts. Your speaker's section contents are stretching. That makes it a bit different from the original design layout.

Your design      |
:-------------------:|
![e1](https://user-images.githubusercontent.com/39523312/147304365-c3cc4b5d-fd5e-48bd-94a4-0b44e754559f.png)

Original Design   |
:---------------------:|
![e2](https://user-images.githubusercontent.com/39523312/147304300-42cba3d6-11b4-40c3-a5a4-ce899c3f9896.png)